documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1519,6 +1517,14 @@ If the captured table(s) schema changes infrequently or never, this is the ideal
 |The number of hours in the past from SYSDATE to mine archive logs.
 Using the default `0` will mine all archive logs.
 
+|[[oracle-property-log-mining-archive-log-only-mode]]<<oracle-property-log-mining-archive-log-only-mode, `+log.mining.archive.log.only.mode+`>>
+|`false`
+|Controls whether or not the connector mines changes from just archive logs or a combination of the online redo logs and archive logs (the default). +
+ +
+For some environments where online redo logs are archived frequently enough to cause LogMiner session failures,
+this opt-in feature only mines archive logs which are guaranteed to be reliable unlike the circular buffer nature of redo logs that can be archived at any point, leading to LogMiner session errors.
+Note that by using this opt-in feature, events will be emitted with a certain amount of latency that is entirely dependent on how frequently online redo logs are archived by Oracle.
```

Comment:
Where is this one coming from?

Reply:
This property is in the main properties table in the version of `oracle.adoc` on `master`.

Reply:
This was added in [DBZ-3561](https://issues.redhat.com/browse/DBZ-3561) to satisfy a situation where redo logs were being moved to archive faster than the connector could keep up and mine them and since the redo logs were named within sequences in their names, i.e. `redoa.log`, `redob.log`, etc that the connector would fail more often than not just due to the speed at which logs rotated.

Reply:
Ok, gotcha. Why is it facing up in this PR, though?